lib/stdlib/src/erl_anno.erl

```diff
@@ -251,24 +251,37 @@ column(Anno) ->
     end.
 
 -doc """
-Returns the end location of the text of the annotations Anno. If there is no
-text, `undefined` is returned.
+Returns the end location of the annotations Anno.
+
+If the end location annotation is present, its value is returned. Otherwise,
+if the text annotation is present, the end location is inferred from the
+location and the text. Finally, if there is no text, `undefined` is returned.
 """.
 -doc(#{since => <<"OTP 18.0">>}).
 -spec end_location(Anno) -> location() | 'undefined' when
       Anno :: anno().
```

Comment:
I think you need to update `-type annotation()` to include `end_location`

Reply:
Good catch, thank you!